tests/Keycloak/CreatedKeycloakHttpClient.php

```diff
@@ -13,7 +13,7 @@
 use GuzzleHttp\RequestOptions;
 use Psr\Log\LoggerInterface;
 
-trait KeycloakHelper
+trait CreatedKeycloakHttpClient
```

Comment:
Maybe the name should include `Factory`? I also notice that `$mock` is always provided?